src/routes/routes.customers.ts

```diff
@@ -54,4 +54,8 @@ export const CustomersRoute = (app: express.Application) => {
   app.put('/:customerID', AccessControlMiddelware._grantAccess((query) => {
     return query.updateOwn(_resource)
   }), CustomersController._updateById);
+
+  app.post('/account/', AccessControlMiddelware._grantAccess((query) => {
```

Comment:
What is that?

Reply:
This endpoint should be called `onboarding/`. 
Please remove the controller right now. The process is really overcomplicated, you haven't take care into the most important detail

Reply:
It's the administrative new customer account creation.

Reply:
Ok. I've readed next comments. I remove it too.